README.md

```diff
@@ -251,6 +252,18 @@ What The Individual Files Do
  - If you select the option to choose your own FFmpeg/mpv optional libraries, this file will contain options that get sent to FFmpeg/mpv's configure script before compiling. Edit this file as you wish to get a smaller FFmpeg/mpv without features you don't need or with additional features not compiled by default, if supported.
 
 
+Optional User Files
+--------
+
+`/local32|64/etc/custom_profile` & `$HOME/custom_build_options`
+ - Put here any general/platform tweaks that you need for _your_ specific environment. For example, if you need to manually set the CUDA_PATH & include in the PATH the binaries for msvc and nvcc, then you can use:
+```
+export CUDA_PATH=$(cygpath -sm "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0")
+export PATH=$PATH:$(dirname "$(cygpath -u "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\bin\Hostx64\x64\cl.exe")")
+export PATH=$PATH:$(dirname "$(cygpath -u "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin\nvcc.exe")")
```

Comment:
I don't like all these paths being absolute. Use variables along with some comments at least. The comments should make it clear that these will break with any update done to both the CUDA SDK and MSVS.